src/script/team/TeamState.ts

```diff
@@ -117,9 +117,7 @@ export class TeamState {
       () => this.teamFeatures()?.mls?.config.protocolToggleUsers.includes(this.userState.self().id) ?? false,
     );
 
-    this.isConferenceCallingEnabled = ko.pureComputed(
-      () => this.teamFeatures()?.conferenceCalling?.status === FeatureStatus.ENABLED,
-    );
+    this.isConferenceCallingEnabled = ko.pureComputed(() => true);
```

Comment:
if it's gonna be always true I suggest a bigger refactor to just get rid of this property every where, but for now we could at least set it to an always true observable maybe

```suggestion
    this.isConferenceCallingEnabled = ko.observable(true);
```

Reply:
O good finding.. ❤️  Thx. this is a hack to activate group call.. This should never go to staging. I will remove this!